api/app/models.py

```diff
@@ -1,21 +1,23 @@
 from typing import List
+import json
 
-from pydantic import BaseModel
+from pydantic import BaseModel, create_model
 from pydantic.class_validators import validator
 from enum import Enum
 
 
-class BaseArticle(BaseModel):
-    id: str
-    abstract: str = None
-    authors: List[str] = []
-    journal: str = None
-    publish_time: str = None
-    title: str
-    source: List[str] = []
-    url: str
+def gettype(name):
+    t = getattr(__builtins__, name)
+    if isinstance(t, type):
+        return t
+    raise ValueError(name)
 
 
+lucene_schema = json.load(open("../lucene_schema.json"))
```

Comment:
put the file name into an env variable, and move all the JSON files into a `schema` folder? then you'll have `acl.json`, `cord19.json` which you could specify in the `.env` file